packages/mobile/src/navigator/Headers.tsx

```diff
@@ -12,23 +12,24 @@ export const nuxNavigationOptions = {
   },
   headerLeftContainerStyle: { paddingHorizontal: 20 },
   headerLeft: <BackButton />,
-  headerRightContainerStyle: { paddingRight: 15 },
-  header: (
-    <View style={{ paddingVertical: 30 }}>
+  headerRightContainerStyle: { paddingHorizontal: 20 },
+  headerRight: <View />,
+  headerTitle: (
+    <View style={{ paddingTop: 5, alignItems: 'center', flex: 1, justifyContent: 'center' }}>
```

Comment:
Nit, could you set these values on `headerTitleContainerStyle` instead of creating a view wrapper here?